language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/grpc/CommandCompletionClientImpl.java

```diff
@@ -16,6 +17,8 @@
 import io.reactivex.Flowable;
 import io.reactivex.Single;
 
+import javax.swing.text.html.Option;
```

Comment:
Wat.

Reply:
Wat.

Reply:
Removing this, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3713/commits/9784a8d5140f0af47e9dd3a540407cfd23b437e6